content/guides/developer-guides/delfin/driver-developer-guide/_index.md

```diff
@@ -0,0 +1,241 @@
+  {
+    "rest": {
+      "host": "127.0.0.1",
+      "port": "22",
+      "username": "username",
+      "password": "password",
+      "vendor": "sample_vendor",
+      "model": "sample_model"
+    }
+  }'
+
+  # Example Response:
+
+  '{"created_at": "2020-09-15T14:10:10.692091", "updated_at": null, "id": "849e0e90-35bd-4d23-8155-53a5a583bcbf", "name": "sample", "vendor": "sample_vendor", "description": "sample", "model": "sample_model", "status": "normal", "serial_number": 123, "firmware_version": "1.0.0", "location": "HK", "total_capacity": 123, "used_capacity": 0, "free_capacity": 123, "raw_capacity": 123, "subscribed_capacity": 123, "sync_status": "SYNCED", "deleted_at": null, "deleted": false}'
+
+
+  curl -H "Content-Type: application/json" -X GET http://localhost:8190/v1/storages
+
+  # Example Response:
+
+  {"storages": [{"created_at": "2020-09-15T14:11:05.611150", "updated_at": "2020-09-15T14:11:05.919163", "id": "2021529b-0235-4a75-b358-0779189d6463", "name": "fake_driver", "vendor": "fake_vendor", "description": "fake driver.", "model": "fake_model", "status": "normal", "serial_number": "1f8d0680-491d-4b13-9b08-996e1ef8ae6d", "firmware_version": "1.0.0", "location": "HK", "total_capacity": 1355, "used_capacity": 934, "free_capacity": 421, "raw_capacity": 2423, "subscribed_capacity": 3884, "sync_status": "SYNCED", "deleted_at": null, "deleted": false}, {"created_at": "2020-09-15T14:10:10.692091", "updated_at": "2020-09-15T14:10:10.970153", "id": "849e0e90-35bd-4d23-8155-53a5a583bcbf", "name": "sample", "vendor": "sample_vendor", "description": "sample", "model": "sample_model", "status": "normal", "serial_number": "123", "firmware_version": "1.0.0", "location": "HK", "total_capacity": 123, "used_capacity": 0, "free_capacity": 123, "raw_capacity": 123, "subscribed_capacity": 123, "sync_status": "SYNCED", "deleted_at": null, "deleted": false}]}
+
+  ```
+* Ensure APIs of list_*() and alert*() works as expected.
+
+  ``` bash
+  # Example delfin APIs that can be used with Sample driver
+  curl -H "Content-Type: application/json" -X GET http://localhost:8190/v1/storages
+  curl -H "Content-Type: application/json" -X GET http://localhost:8190/v1/storages-pools
+  curl -H "Content-Type: application/json" -X GET http://localhost:8190/v1/volumes
+  curl -X POST http://localhost:8190/v1/storages/<storage_id>/sync
+  ...
+  ```
+
+* Raise PR with test reports to Delfin repository.
+
+### How it works
+
+Third party drivers are located and loaded into Delfin, when the create storage API (POST request) is invoked for that backend.
+
+Create Storages POST API contains a request body model namely access_info, which contains fields ‘vendor’ & ‘model’. These fields are used to match and identify the driver from the ‘entry_points’ registered by the driver. Delfin loads this matched driver and registers the driver for the created backend. Any further API calls on this backend will use this newly loaded driver.
```

Comment:
nit: use inline code highlighting to highlight field names.
'vendor' -> `vendor`
'model' -> `model`